neqo-transport/src/lib.rs

```diff
@@ -31,15 +31,19 @@ pub mod tparams;
 mod tracking;
 
 pub use self::cid::{ConnectionId, ConnectionIdManager};
-pub use self::connection::{Connection, FixedConnectionIdManager, Output, State, ZeroRttState};
+pub use self::connection::{
+    Connection, DraftVersion, FixedConnectionIdManager, Output, State, ZeroRttState,
+};
 pub use self::events::{ConnectionEvent, ConnectionEvents};
 pub use self::frame::CloseError;
 pub use self::frame::StreamType;
 pub use self::stream_id::StreamId;
 
 /// The supported version of the QUIC protocol.
 pub type Version = u32;
-pub const QUIC_VERSION: Version = 0xff00_0000 + 27;
+
+/// Supporting more than one version, it's helpful to have a default.
+pub const DEFAULT_QUIC_VERSION: Version = 0xff00_0000 + 28;
```

Comment:
Maybe instead

```rust
enum Version {
Draft27,
Draft28,
}
impl Default for Version {
fn default() -> Self {
  Self::Draft28
}
}
```

Reply:
I don't think we need the default, actually..